python/ccf/ledger.py

```diff
@@ -19,6 +29,12 @@
 
 UNPACK_ARGS = {"raw": True, "strict_map_key": False}
 
+# Public table names as defined in CCF
+# https://github.com/microsoft/CCF/blob/main/src/node/entities.h#L96
```

Comment:
Minor: better not to reference an exact line in the C++ source code as it is likely to change in the future.

Reply:
makes sense